examples/plugins/StringChecker.php

```diff
@@ -13,10 +13,6 @@ class StringChecker implements AfterExpressionAnalysisInterface
 {
     /**
      * Called after an expression has been checked
-     *
-     * @param  FileManipulation[]   $file_replacements
-     *
-     * @return null|false
```

Comment:
You can leave null|false, it's more narrow than ?bool

Reply:
Thanks for pointing this out. Left null|false here and in the other file